### PR TITLE
Update env to add geemap, gh and is2view

### PR DESCRIPTION
**Pull request by tsnow03**

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -66,6 +68,7 @@ dependencies:
   - matplotlib==3.6.2
   - plotly==5.11.0
   - seaborn==0.12.1 # statistical plotting with matplotlib
+  - geemap==0.24.4
```
Comment (weiji14) [inline environment.yml:71]:
```suggestion
  - geemap~=0.24.4
```

File: environment.yml [hunk 2]
```diff
@@ -23,6 +23,8 @@ dependencies:
   # https://blog.jupyter.org/securely-pushing-to-github-from-a-jupyterhub-3ee42dfdc54f has
   # some more information
   - gh-scoped-creds==4.1
+  # More git tools from github-cli
+  - gh
```
Comment (weiji14) [inline environment.yml:27]:
```suggestion
  - gh~=2.32.1
```

**Review by weiji14 — COMMENTED** [review]
Some suggestions to follow the compatibility release spec `~=` as per #46. Shouldn't need to relock the conda-lock file afterwards unless the versions have changd in <1 hour.

**Comment by weiji14** [comment]
@tsutterley requested that we add https://anaconda.org/conda-forge/is2view to the hub-image, ok if I do it in this PR too @tsnow03?

**Comment by tsnow03** [comment]
Yes! Thanks @weiji14

**Review by weiji14 — APPROVED** [review]
Should be ok to merge once the tests pass. :rocket:

**Pull request merged**